src/kernel-install/kernel-install

```diff
@@ -128,10 +128,18 @@ fi
 
 ENTRY_DIR_ABS="$BOOT_ROOT/$MACHINE_ID/$KERNEL_VERSION"
 
+# Provide a directory where to store generated initrds
+cleanup() {
```

Comment:
Where is this used??

Reply:
Ooops, there was supposed to be `trap` line for this.
/me checks, and indeed there are 500 snapshot directories in /tmp